_extras/guide.md

```diff
@@ -30,7 +30,7 @@ In order to edit your command prompt, type `PS1='$ '` into your shell and press
 
 In order to reset the command prompt, type `source .bashrc` in order to source the bash profile, or type `PS1="\u@\h:\w $ "` in order to set the prompt to show username, "@", hostname, ":", and current working directory (ie. the user's current location within the filesystem). 
 
-NOTE: Editing the prompt is discussed in [lesson 01 - Introduction](https://datacarpentry.org/shell-genomics/01-introduction/index.html) under the 'Navigating your file system' section. This explains how to edit the prompt via `PS1='$ '` as here, so it would perhaps be best to *start* the lesson with the *default* prompt (as all the learners will and they can see that their screen will reassuringly match the instructor's screen at this point), and then instructors can choose to edit their prompt and talk through how they're doing that for learners' benefit at this section, or the instructor can just make the change early in the lesson for the visibility benefit, and explain to learners that they can find out how to do this in the lesson materials. 
+NOTE: Editing the prompt is discussed in [1. Introducing the Shell](https://datacarpentry.org/shell-genomics/01-introduction/index.html) under the 'Navigating your file system' section. This explains how to edit the prompt via `PS1='$ '` as here, so it would perhaps be best to *start* the lesson with the *default* prompt (as all the learners will and they can see that their screen will reassuringly match the instructor's screen at this point), and then instructors can choose to edit their prompt and talk through how they're doing that for learners' benefit at this section, or the instructor can just make the change early in the lesson for the visibility benefit, and explain to learners that they can find out how to do this in the lesson materials. 
```

Comment:
```suggestion
NOTE: Editing the prompt is discussed in [1. Introducing the Shell](https://datacarpentry.org/shell-genomics/01-introduction/index.html#navigating-your-file-system). This explains how to edit the prompt via `PS1='$ '` as here, so it would perhaps be best to *start* the lesson with the *default* prompt (as all the learners will and they can see that their screen will reassuringly match the instructor's screen at this point), and then instructors can choose to edit their prompt and talk through how they're doing that for learners' benefit at this section, or the instructor can just make the change early in the lesson for the visibility benefit, and explain to learners that they can find out how to do this in the lesson materials. 
```